REGRESSION(257006@main): [BigSur+ wk1 EWS ] 2X imported/w3c/web-platform-tests/geolocation-API (layout-tests) are constant text failures

marcoscaceres opened the pull request:
#### dec2d08b366ddef9bdef9aa8ffb68f1ec806ff4a
<pre>
REGRESSION(257006@main): [BigSur+ wk1 EWS ] 2X imported/w3c/web-platform-tests/geolocation-API (layout-tests) are constant text failures
<a href="https://bugs.webkit.org/show_bug.cgi?id=248517">https://bugs.webkit.org/show_bug.cgi?id=248517</a>
rdar://102798964

Reviewed by NOBODY (OOPS!).

Resynced with WTPT upstream to 1d55076317.

In <a href="https://github.com/WebKit/WebKit/pull/6845">https://github.com/WebKit/WebKit/pull/6845</a>, granting permission for geolocation
only emits a single position object, which should reduce the flakyness.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/disabled-by-feature-policy.https.sub-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/disabled-by-feature-policy.https.sub.html.headers: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/disabled-by-permissions-policy.https.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/disabled-by-permissions-policy.https.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/geolocation-API/disabled-by-feature-policy.https.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/disabled-by-permissions-policy.https.sub.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy-attribute-redirect-on-load.https.sub-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy-attribute.https.sub-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy.https.sub-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy.https.sub.html.headers: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-permission-policy-attribute-redirect-on-load.https.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-permission-policy-attribute-redirect-on-load.https.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy-attribute-redirect-on-load.https.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-permission-policy-attribute.https.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-permission-policy-attribute.https.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy-attribute.https.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-permissions-policy.https.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-permissions-policy.https.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy.https.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-permissions-policy.https.sub.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-on-self-origin-by-feature-policy.https.sub-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-on-self-origin-by-feature-policy.https.sub.html.headers: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-on-self-origin-by-permissions-policy.https.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-on-self-origin-by-permissions-policy.https.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-on-self-origin-by-feature-policy.https.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-on-self-origin-by-permissions-policy.https.sub.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/getCurrentPosition_permission_allow.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/getCurrentPosition_permission_allow.https.html:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/getCurrentPosition_permission_deny.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/watchPosition_permission_deny.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/permissions-policy/resources/permissions-policy-geolocation.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/feature-policy/resources/feature-policy-geolocation.html.
* LayoutTests/imported/w3c/web-platform-tests/permissions-policy/resources/permissions-policy.js: Added.
(assert_permissions_policy_supported):
(test_feature_availability):
(expect_feature_available_default):
(expect_feature_unavailable_default):
(test_feature_availability_with_post_message_result):
(async test_feature_in_iframe):
(same_origin_url):
(cross_origin_url):
(run_all_fp_tests_allow_self):
(expect_reports):
* LayoutTests/imported/w3c/web-platform-tests/permissions-policy/resources/redirect-on-load.html: Added.
* LayoutTests/platform/mac-wk1/TestExpectations:
</pre><!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/dec2d08b366ddef9bdef9aa8ffb68f1ec806ff4a

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [❌ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/98943 "Failed to checkout and rebase branch from PR 7197") | [❌ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/8152 "Failed to checkout and rebase branch from PR 7197") | [  ~~🛠 mac~~](https://ews-build.webkit.org/#/builders/43/builds/32104 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [❌ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/108355 "Failed to checkout and rebase branch from PR 7197") | [❌ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/168610 "Failed to checkout and rebase branch from PR 7197") 
| | [❌ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/8705 "Failed to checkout and rebase branch from PR 7197") | [  ~~🛠 mac-AS-debug~~](https://ews-build.webkit.org/#/builders/61/builds/85507 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 gtk~~](https://ews-build.webkit.org/#/builders/36/builds/91475 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/12/builds/106324 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [❌ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/104643 "Failed to checkout and rebase branch from PR 7197") | [❌ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/76/builds/8705 "Failed to checkout and rebase branch from PR 7197") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/43/builds/32104 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/91475 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [❌ 🧪 api-ios](https://ews-build.webkit.org/#/builders/76/builds/8705 "Failed to checkout and rebase branch from PR 7197") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/43/builds/32104 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/36/builds/91475 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/81/builds/2057 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/43/builds/32104 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/82/builds/1963 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/61/builds/85507 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [❌ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/6923 "Failed to checkout and rebase branch from PR 7197") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/43/builds/32104 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [❌ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/3370 "Failed to checkout and rebase branch from PR 7197") | | | 
<!--EWS-Status-Bubble-End-->

youennf reviewed (approved):
r=me once bots are happy

marcoscaceres committed:
REGRESSION(257006@main): [BigSur+ wk1 EWS ] 2X imported/w3c/web-platform-tests/geolocation-API (layout-tests) are constant text failures
https://bugs.webkit.org/show_bug.cgi?id=248517
rdar://102798964

Reviewed by NOBODY (OOPS!).

Resynced with WTPT upstream to 1d55076317.

In https://github.com/WebKit/WebKit/pull/6845, granting permission for geolocation
only emits a single position object, which should reduce the flakyness.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/disabled-by-feature-policy.https.sub-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/disabled-by-feature-policy.https.sub.html.headers: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/disabled-by-permissions-policy.https.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/disabled-by-permissions-policy.https.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/geolocation-API/disabled-by-feature-policy.https.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/disabled-by-permissions-policy.https.sub.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy-attribute-redirect-on-load.https.sub-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy-attribute.https.sub-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy.https.sub-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy.https.sub.html.headers: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-permission-policy-attribute-redirect-on-load.https.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-permission-policy-attribute-redirect-on-load.https.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy-attribute-redirect-on-load.https.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-permission-policy-attribute.https.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-permission-policy-attribute.https.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy-attribute.https.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-permissions-policy.https.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-permissions-policy.https.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy.https.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-permissions-policy.https.sub.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-on-self-origin-by-feature-policy.https.sub-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-on-self-origin-by-feature-policy.https.sub.html.headers: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-on-self-origin-by-permissions-policy.https.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-on-self-origin-by-permissions-policy.https.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-on-self-origin-by-feature-policy.https.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-on-self-origin-by-permissions-policy.https.sub.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/getCurrentPosition_permission_allow.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/getCurrentPosition_permission_allow.https.html:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/getCurrentPosition_permission_deny.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/watchPosition_permission_deny.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/permissions-policy/resources/permissions-policy-geolocation.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/feature-policy/resources/feature-policy-geolocation.html.
* LayoutTests/imported/w3c/web-platform-tests/permissions-policy/resources/permissions-policy.js: Added.
(assert_permissions_policy_supported):
(test_feature_availability):
(expect_feature_available_default):
(expect_feature_unavailable_default):
(test_feature_availability_with_post_message_result):
(async test_feature_in_iframe):
(same_origin_url):
(cross_origin_url):
(run_all_fp_tests_allow_self):
(expect_reports):
* LayoutTests/imported/w3c/web-platform-tests/permissions-policy/resources/redirect-on-load.html: Added.
* LayoutTests/platform/mac-wk1/TestExpectations:
